.github/workflows/image-optimiser.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   build:
     # Only run on non-draft PRs within the same repository.
     if: github.event.pull_request.head.repo.full_name == github.repository && github.event.pull_request.draft
-      == false
+      == false == false
```

Comment:
is this a typo or it is intended condition?

Reply:
Yeah some copy-paste mistake there. Good catch!